src/components/inputs/textField/textField.component.tsx

```diff
@@ -112,74 +117,150 @@ export const TextField: FC<TextFieldProps> = React.forwardRef(
       rows,
       minRows,
       maxRows,
+      labelLeft,
     },
     ref
   ) => {
     return (
       <FormControl className={styles.formControl} fullWidth variant="outlined">
-        {label && (
-          <MuiInputLabel
-            variant="standard"
-            className={styles.label}
-            htmlFor={id}
-            disableAnimation
-            shrink
-            required={required}
+        {label && labelLeft ? (
+          <Box
             sx={{
-              color: 'common.black',
-              '&.Mui-focused': {
-                color: 'common.black',
-              },
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'flex-end',
             }}
           >
```

Comment:
Please don't use `Box` and `sx`. Use a `div` and some class instead. Or am I missing something?

Reply:
Changed it to a div!